src/lib/Navigation.svelte

```diff
@@ -1,4 +1,83 @@
 <script>
+	import NavDropdownMenu from '../lib/NavDropdownMenu.svelte';
+
+	const navigations = [
+		{
+			name: 'Tools',
+			items: [
+				{
+					name: 'Looker',
+					url: 'Looker',
+					description: `Mozilla's primary tool for analyzing data.`
+				},
+				{
+					name: 'Glean Aggregated Metrics Dashboard (GLAM)',
+					url: 'glam.telemetry.mozilla.org',
+					description: `An interactive dashboard that examines the distributions of values of specific individual telemetry metrics, over various dimensions.`
+				},
+				{
+					name: 'Experimenter',
+					description: `The central resource for A/B experiments and feature rollouts in Firefox Mobile and Desktop`,
```

Comment:
Keep the same order of properties for consistency (i.e., swap `description` and `url` here because all the others are that order).